test/verify/check-storage-hidden

```diff
@@ -94,19 +93,19 @@ class TestStorageHidden(StorageCase):
 
         self.login_and_go("/storage")
 
-        m.add_disk("50M", serial="DISK1")
-        m.add_disk("50M", serial="DISK2")
-        b.wait_in_text("#drives", "DISK1")
-        b.wait_in_text("#drives", "DISK2")
+        disk1 = self.add_loopback_disk()
+        disk2 = self.add_loopback_disk()
+        b.wait_in_text("#others", disk1)
+        b.wait_in_text("#others", disk2)
 
         # Now do the same with a MDRAID
 
         self.dialog_with_retry(trigger=lambda: self.devices_dropdown('Create RAID device'),
```

Comment:
Same as above, this probably needs a cleanup function to remove the vgroup on test failure.

Reply:
I did the same check on `TestStorageHidden.testHiddenRaid`, and *that* indeed gets stuck .. /dev/md/ARR still exists, and `losetup -a` says
```
/dev/loop1: [0031]:221217 (/var/tmp/loop.BkIX (deleted))
/dev/loop0: [0031]:221206 (/var/tmp/loop.TaNd (deleted))
```
after the cleanup. That's because removing `scsi_debug` has enough force to tear down all devices on top of it, but mere loop devices don't. So indeed this needs an opportunistic cleanup. I'm not actually sure how to clean that up -- it's not LVM, nor is it in `dmsetup ls`. It's plain `mdadm`, but that doesn't actually have a "teardown/remove/disassemble" command. There is no existing cleanup in our tests for that, probably because test/verify/check-storage-mdraid is destructive.

I tried
```
# mdadm /dev/md127 --stop
mdadm: No action given for /dev/md127 in --misc mode
       Action options must come before device names

# mdadm --manage /dev/md127 --stop
mdadm: Cannot get exclusive access to /dev/md127:Perhaps a running process, mounted filesystem or active volume group?
```

This is too hard/brittle, and not worth it -- I'll just make the Raid test destructive again.

Reply:
FTR, I was too curious and tried this with a RAM disk. It doesn't work either ("rmmod: ERROR: Module scsi_debug is in use"), so I'll drop the commit message comment about "in the future we may..." and just leave it at that.